kotlin/internal/opts.bzl

```diff
@@ -13,6 +13,13 @@
 # limitations under the License.
 
 _KOPTS = {
+    "all_warnings_as_errors": struct(
```

Comment:
You might want to keep the consistent naming pattern used above, so this would be `w_error`

Reply:
I have to say, I'm not sure I get the point of this abstraction if it doesn't provide a higher-level API over the compiler configuration (the fact that they directly map to flags isn't especially relevant IMHO). I'm happy to rename it for the sake of consistency, but I don't think it's the best approach here. As it is, you would have this configuration:

```
kt_kotlinc_options(
  name = "toolchain_options",
  all_warnings_as_errors = True
)
```

vs:

```
kt_kotlinc_options(
  name = "toolchain_options",
  w_error = True
)
```

I personally find the first one clearer and more obvious.